docs/installation/installing-with-npm.md

```diff
@@ -1,303 +1,196 @@
+- add GOV.UK Frontend's JavaScript file to your HTML
+- import the JavaScript using a bundler like [Webpack](https://webpack.js.org/)
 
-If you're using a bundler such as Webpack, use the `import` syntax to import a component:
+You must also initialise GOV.UK Frontend's components by calling the `initAll` function. If you do not initialise, some components will not work correctly for users who have accessible needs or use older browsers.
 
-```JS
-import { Radios } from 'govuk-frontend'
-```
+### Add the JavaScript file to your HTML
 
-If you're using a bundler such as [Browserify](http://browserify.org/), you may need to use the CommonJS `require`:
+First either:
 
+- route requests to the `node_modules/govuk-frontend/govuk/all.js` file
+- copy the file into your project folder
 
-```JS
-const GOVUKFrontend = require('govuk-frontend')
+Then add GOV.UK Frontend's Javascript file before the closing `</body>` tag of your HTML page or page template, and run the `initAll` function to initialise all the components.
 
-const Radios = GOVUKFrontend.Radios
+```html
+<body>
+...
+  <script src="node_modules/govuk-frontend/govuk/all.js"></script>
+  <script>
+    window.GOVUKFrontend.initAll()
+  </script>
+</body>
 ```
 
-GOV.UK Frontend components with JavaScript behaviour have the `data-module` attribute set in their markup.
+You can also select specific components, then call `initAll` with a `scope` parameter that includes only those components. For example, to initialise only [radios](https://design-system.service.gov.uk/components/radios/):
 
-You can use this attribute to initialise the component manually, this may be useful if you are adding markup to a page after it has loaded.
-
-To initialise the first radio component on a page, use:
-
-```js
-var $radio = document.querySelector('[data-module="govuk-radios"]')
-if ($radio) {
-  new Radios($radio).init()
-}
+```html
+...
+  <script>
+    var radio = document.querySelector('.radio')
+    window.GOVUKFrontend.initAll({
+      scope: $radio
+    })
+  </script>
 ```
 
-### Polyfills
-A JavaScript polyfill provides functionality on older browsers or assistive technology that do not natively support it.
-
-The polyfills provided with GOV.UK Frontend aim to fix usability and accessibility issues. If there is a JavaScript included in the component directory, it is important to import and initialise it in your project to ensure that all users can properly use the component (see [Polyfilling](/docs/contributing/polyfilling.md)).
+If you add markup to a page after it's loaded, you can also use the `data-module` attribute to select specific components. For example, you can select radios using their `data-module` attribute of `govuk-radios`.
 
-### How GOV.UK Frontend is bundled
-The JavaScript included in GOV.UK Frontend components are in [UMD (Universal Module Definition)](https://github.com/umdjs/umd) format which makes it compatible with AMD (Asynchronous module definition) and CommonJS.
```

Comment:
Would it be worth saying this somewhere else as it's deleted from here? Some users might find this useful when deciding which bundler to use.

Reply:
I spoke to @nickcolley about this and we did keep a version of it at one point, but if I remember right we chose to remove it?